test/integration/multiplexed_integration_test.cc

```diff
@@ -864,6 +880,9 @@ TEST_P(Http2IntegrationTest, CodecErrorAfterStreamStart) {
 }
 
 TEST_P(Http2IntegrationTest, BadMagic) {
+  if (downstreamProtocol() == Http::CodecClient::Type::HTTP3) {
+    return; // protocol-specific framing.
+  }
```

Comment:
EXCLUDE_DOWNSTREAM_HTTP3 means need to fix. This one doesn't?

Reply:
correct.  I don't think H3 has the equivalent of the initial H2 magic, so we don't have to replicate this test for H3.
I'll add a better comment next I'm in that file.

Reply:
Or just rename the test BadHttp2Magic